### PR TITLE
[UI]: Display filename after upload.

### PR DESCRIPTION
**Pull request by SAEb-ai**
Signed-off-by: Md Shahbaz Alam <shahbazalam75508@gmail.com>

**Description**

This PR fixes https://github.com/meshery/meshery/issues/5345

**Notes for Reviewers**
Video Illustration:

https://user-images.githubusercontent.com/69013647/167778682-50e83305-e271-48a3-a97f-e08749b9a4bf.mp4

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Review by MUzairS15 — DISMISSED** [review]
LGTM, Thanks

**Review by Abhijay007 — APPROVED** [review]
LGTM

**Review by leecalcote — CHANGES_REQUESTED** [review]
Good one, @SAEb-ai 👍 

Will you have a look at the failing lint check?

**Comment by SAEb-ai** [comment]
> Good one, @SAEb-ai 👍
> 
> Will you have a look at the failing lint check?

Thanks @leecalcote !! I will try to resolve the failure of lint checks by tommorow!!

**Comment by SAEb-ai** [comment]
@leecalcote Please take a look. I have fixed the lint failures. Thanks!!

File: ui/components/MeshConfigComponent.js [hunk 1]
```diff
@@ -140,7 +140,7 @@ async function uploadK8sConfig() {
   const field = document.getElementById("k8sfile");
   if (field instanceof HTMLInputElement) {
     if (field.files.length < 1) return;
-
+    const name=field.files[0].name;
```
Comment (adithyaakrishna) [inline ui/components/MeshConfigComponent.js:143]:
```suggestion
    const name = field.files[0].name;
```

Comment (adithyaakrishna) [reply ui/components/MeshConfigComponent.js:143]: @SAEb-ai Could you please make this small change? Other than that, all others LGTM 🎉

**Review by adithyaakrishna — CHANGES_REQUESTED** [review]

**Comment by SAEb-ai** [comment]
@leecalcote Can you please review this and merge this for me? Thanks!!

File: ui/components/MeshConfigComponent.js (hunk 1 above)
Comment (SAEb-ai) [reply ui/components/MeshConfigComponent.js]: Done

**Review by adithyaakrishna — APPROVED** [review]

**Review by warunicorn19 — APPROVED** [review]
Thank you @SAEb-ai

**Comment by leecalcote** [comment]
@SAEb-ai! 👏 😄

**Pull request merged**